### DISCUSSION TITLE
Custom /+1w teamplate for note

### DISCUSSION THREAD

**Discussion by fyod0r1** [foambubble/foam#1350] [Q&A]
Is it possible to-do such note name/path: 
FOAM_DATE_WEEK = 18
FOAM_DATE_YEAR = 2024
====> note_name = 2024-19
i try different variations of:
---
foam_template:
  filepath: '$FOAM_DATE_YEAR-${FOAM_DATE_WEEK + 1}.md'
type: week_note 
---

**Comment by riccardoferretti** [comment]
We use the same syntax as [VS Code Snippet Variables](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_snippet-syntax) so that is not supported.
I completely agree with the use case though, I have been thinking about ways to make templates more dynamic in that regard (but will likely use a separate approach)
